CRM/Financial/BAO/FinancialItem.php

```diff
@@ -286,25 +286,23 @@ public static function checkContactPresent($contactIds, &$error) {
   }
 
   /**
-   * Get last financial item data.
+   * Get non-liability financial item with the highest id relating to the line item.
    *
-   * @param int $entityId
+   * In practical terms this means return the row that is not associated with
+   * sales tax as sales tax is a liability account.
    *
-   * @param string $entityTable
+   * @param int $entityId
    *
    * @return object CRM_Core_DAO
    */
-  public static function getPreviousFinancialItem($entityId, $entityTable = 'civicrm_line_item') {
```

Comment:
Actually I strongly feel we should not support extensions accessing 'inner functions' like this - it would make us responsible for their breakage if we change the signature or any other aspect. If extensions need this we need to figure out how to expose through the api